cmd/acra-server/acra-server.go

```diff
@@ -277,25 +279,31 @@ func main() {
 			log.WithError(err).Errorln("Can't initialize identifier converter")
 			os.Exit(1)
 		}
-		tlsWrapper, err := network.NewTLSAuthenticationConnectionWrapper(clientTLSConfig, dbTLSConfig, network.CommonNameExtractor{}, idConverter)
+		tlsWrapper, err = network.NewTLSAuthenticationConnectionWrapper(dbTLSConfig, clientTLSConfig, network.CommonNameExtractor{}, idConverter)
```

Comment:
because I mismatched order and tests failed because dbTLSConfig configured with SNI which verified in connection to db and useless for wrapping connection with app